FelixKlauke/paperspigot-docker
# Release (#58)

* Update .travis.yml

* Update .travis.yml

* Update README.md

* Use latest version

* Update for v1.15.2 (#42)

* Add start_period in compose files to avoid failing health checks at start (#44)

* Update README.md

Disregard failing health checks during start period (of 60 sec). Prevents container from being marked as 'unhealthy' and then restarted when deployed via docker swarm.

* Update Dockerfile

Disregard failing health checks during start period (of 60 sec). Prevents container from being marked as 'unhealthy' and then restarted when deployed via docker swarm.

* Update README.md

moved health check start-period to Dockerfile

* Hotfix java version not used in build args

* Create dockerimage.yml

* Fix typo

* Updated for 1.16 (#51)

* Updated for 1.16

* Update .travis.yml

* Update docker-compose.yml

* Update README.md

* Fix: Server not shutting down gracefully; minor updates and changes (#54)

* Updated README.md; fixed some grammar and spelling mistakes, added "see also" section

* Updated PAPER_VERSION argument to 1.16.1 for the sake of consistency

* [FIX] Java not receiving SIGTERM: Starting the server through the exec command now allows it to correctly receive and respond to system signals

* Add 1.16.2 and 1.16.3. 

Fixed #57

Co-authored-by: Aaron <44198148+whalehub@users.noreply.github.com>
Co-authored-by: oschlueter <10252511+oschlueter@users.noreply.github.com>
Co-authored-by: Avior <florian.bouillon@delta-wings.net>
Co-authored-by: Max <max.carrara@austrianaudio.com>
